.github/workflows/tests.yml

```diff
@@ -39,7 +57,7 @@ jobs:
 
       - name: Install package
         run: |
-          pip install -e ".[test]"
+          pip install -e ".[test, redis]"
```

Comment:
It might be good to run the tests with and without `redis`. Maybe have a `make build-with-redis`

Reply:
@wesleybl All tests are automatically executed with and without redis 😄  See https://github.com/collective/collective.elasticsearch/blob/mle-redis-rq/src/collective/elasticsearch/tests/test_processor.py#L15-L20